services/capture/src/main/java/ca/bc/gov/tno/services/capture/config/CaptureConfig.java

```diff
@@ -51,6 +51,16 @@
    */
   private long streamStartTime;
 
+  /**
+   * The timezone of the capture service
+   */
+  private String timeZone;
+
+  /**
+   * The command executing this capture process
```

Comment:
Description doesn't help me understand the variable regrettably.

Reply:
Changed it to "The curl or ffmpeg command executing this capture process"